src/lib/Repository/UiRelationService.php

```diff
@@ -62,19 +61,16 @@ public function loadRelations(VersionInfo $versionInfo)
     /**
      * Loads reverse relations for a piece of content.
      * Retrieves reverse relations and then sets the field definition and the content type name.
-     * Returns false if the user doesn't have permission to load reverse relations.
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\UnauthorizedException if the user is not allowed to read reverse relations.
      *
      * @param ContentInfo $contentInfo
      *
-     * @return UiRelation[]|bool
+     * @return UiRelation[]
      */
     public function loadReverseRelations(ContentInfo $contentInfo)
     {
-        try {
```

Comment:
I don't like the idea of returning a constant or an array from within the method as it's not expected behaviour. I think keeping this generic without checking for permissions may be better as there may come a time when we need the reverse relations and it should throw an exception if someone doesn't have access. 
Thoughts?